geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1378,6 +1379,18 @@
    */
   String NAME = "name";
   /**
+   * The authentication mode for the protobuf client-server protocol.
+   *
+   * </p>
+   * <U>Description</U>: Specifies the authentication mode used by the geode-protobuf module.
+   * </p>
+   * <U>Default</U>: "NOOP"
+   * </p>
+   * <U>Allowed values</U>: "NOOP" "SIMPLE"
+   */
+  @Experimental
+  String PROTOBUF_PROTOCOL_AUTHENTICATION_MODE = "protobuf-protocol-authentication-mode";
```

Comment:
This property is misleading. It is NOT a protobuf specific authentication mode. It is merely an authentication mechanism that uses protobuf underneath the covers.
1) A different property name is to be used
2) With this property, the feature toggle should also maybe be removed??!!? One cannot live without the other